docs/core/tools/dotnet-new.md

```diff
@@ -33,28 +32,28 @@ After this, the project is ready to be compiled and/or edited further.
 
 ## Options
 
-`-l`, `--lang [C#|F#]`
+`-h|--help`
 
-Language of the project. Defaults to `C#`. `csharp` (`fsharp`) or `cs` (`fs`) are also valid options.
+Prints out a short help for the command.  
+
+`-l|--lang <C#|F#>`
+
+Language of the project. Defaults to `C#`. Valid values are `csharp`, `fsharp`, `cs` and `fs`.
```

Comment:
I think the "also" in the previous version was important. Either this should say something like "additional valid values", or it should list all valid values, including `C#` and `F#`.